catalog/app/containers/Admin/selectors.js

```diff
@@ -0,0 +1,36 @@
+import { Map } from 'immutable';
+import { createSelector } from 'reselect';
+
+/**
+ * Direct selector to the admin state domain
+ */
+export const selectAdminDomain = (state) => state.get('admin', Map({}));
+
+/**
+ * Other specific selectors
+ */
+
+/**
+ * Default selector used by Admin
+ */
+
+export default createSelector(
+  selectAdminDomain,
+  (substate) =>
+    substate
+      .updateIn(['members', 'response'], (members) =>
+        members && members.filter
+          ? members
+            .filter((m) => m.get('status') !== 'disabled')
+            .sortBy((m) => m.get('name'))
+          : members
+      ) // eslint-disable-line function-paren-newline
+      .updateIn(['packages', 'response'], (packages) =>
+        packages && packages.filter
+          ? packages
+            .filter((m) => m.get('deletes') === 0)
```

Comment:
@akarve should packages be filtered like this?

Reply:
Good question. No. And only because there's no delete flag right now.

Reply:
k